src/modules/job-manager/kill.c

```diff
@@ -72,9 +72,7 @@ void kill_handle_request (flux_t *h,
     int sig;
     flux_future_t *f;
     char topic [64];
-    const char *errstr = NULL;
-
-    if (flux_request_unpack (msg, NULL, "{s:I s:i}",
+    const char *errstr = NULL; if (flux_request_unpack (msg, NULL, "{s:I s:i}",
```

Comment:
not sure what happened here :-)

Reply:
oops, probably accidentally joined two lines while editing. Thanks!